src/api/test/unit/code_quality_test.rb

```diff
@@ -91,6 +91,7 @@ def setup
       'SearchController#search'                                                 => 81.15,
       'SourceController#project_command_copy'                                   => 133.93,
       'SourceController#update_project_meta'                                    => 98.09,
+      'User#requests'                                                           => 95.51,
```

Comment:
because this method was calling the horrible `BsRequest#collection` method before, so was just one call :see_no_evil: